src/polymer/polymer-element.ts

```diff
@@ -183,7 +183,15 @@ export class ScannedPolymerElement extends ScannedElement implements
     addMethod(this, method);
   }
 
+  applyDescriptionAsJsdoc() {
+    this.jsdoc = jsdoc.parseJsdoc(this.description);
+    this.description = this.jsdoc.description;
+  }
+
   resolve(document: Document): PolymerElement {
+    if (!this.jsdoc) {
+      this.applyDescriptionAsJsdoc();
+    }
     this.applyJsdocDemoTags(document.url);
```

Comment:
shouldn't this all go in the constructor? I'd like to not mutate the scanned element even in resolve()